matrix-meetings-widget/src/components/meetings/MeetingsCalendar/MeetingsCalendar.test.tsx

```diff
@@ -477,7 +477,7 @@ describe('<MeetingsCalendar/>', () => {
     ).toBeInTheDocument();
     expect(
       within(day1).getByRole('button', {
-        name: 'Show 2 more events',
```

Comment:
Not sure where this is originating from but this feels like a step backwards. The old text had more information i feel like as it indicated being an Action to take.

Reply:
:shrug: seems like a Mui component changed. I can add it as a task to the post migration ticket.